docs/content/backup_restore.md

```diff
@@ -1,9 +1,129 @@
 # Backup and Restore
 
-WIP ...
+The *DbToolsBundle* comes with two Symfony Console commands to backup and restore
+your database but also a tiny backups manager (`DbToolsStorage`) which handle backup
+files for you.
 
 ## Backup command
 
+The backup command will use [defined binary]('./configuration#binaries') for your database vendor with correct parameters
```

Comment:
Pardon my french.

Reply:
configured ?

Reply:
I'm not sure about what you are trying to say, are they "predefined" as you pre-configured them, are they "configured" as the user did configure it, or they just are because it's hardcoded ?

Reply:
I updated with `predefined or configured binary`